webpack/assets/javascripts/react_app/routes/common/reducerHOC/withQueryReducer.js

```diff
@@ -0,0 +1,22 @@
+import Immutable from 'seamless-immutable';
+
+const initialState = Immutable({
```

Comment:
@tbrisker initial state won't include a `perPage`. pages should fetch it from URI/Redux